### PR TITLE
New wget Cheat sheet

### PR DESCRIPTION
**Pull request by pa1pal**
IA : [Wget Cheat sheet](https://duck.co/ia/view/wget_cheat_sheet)

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pa1pal Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
## [Wget Cheat Sheet](https://duck.co/ia/view/wget_cheat_sheet)

**Description**: GNU Wget (or just Wget, formerly Geturl) is a computer program that retrieves content from web servers. It is part of the GNU Project. Its name is derived from World Wide Web and get. It supports downloading via HTTP, HTTPS, and FTP protocols.

**Example Query**: [](https://beta.duckduckgo.com/?q=), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: https://www.gnu.org/manual/manual.html

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/wget_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/wget_cheat_sheet)*

File: share/goodie/cheat_sheets/json/wget.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "name": "Wget",
+    "description": "GNU Wget (or just Wget, formerly Geturl) is a computer program that retrieves content from web servers. It is part of the GNU Project. Its name is derived from World Wide Web and get. It supports downloading via HTTP, HTTPS, and FTP protocols.",
+
+    "metadata": {
+        "sourceName": "GNU",
+        "sourceUrl" : "https://www.gnu.org/manual/manual.html"
+    },
+
+    "aliases": [
+        "wget", "wget cheat sheet"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/wget.json]: Please remove these aliases. `wget` is already the default trigger` :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Wget Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wget_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@pa1pal thanks again for all the hard work you've put into creating this Wget Cheat Sheet. You've done a great job so far! We at DuckDuckGo, and the DuckDuckHack community really appreciate you taking the time to contribute.

It looks like this is almost done?  If you're stuck or need any help, I'd love to chat if you're around this week or next. Please message me on slack (@moollaza) or send me an email (moollaza@duckduckgo.com) if you'd be interested in talking. 

Thanks again!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Wget Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wget_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Wget Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wget_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@pa1pal Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Wget Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wget_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Wget Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wget_cheat_sheet_mobile.png)

**Comment by pa1pal** [comment]
Thanks @moollaza . I have some problem with codio I thought subscription was over. But now I am working and interested in finishing this up.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Wget Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wget_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Wget Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wget_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/wget.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    "name": "Wget",
+    "description": "GNU Wget (or just Wget, formerly Geturl) is a computer program that retrieves content from web servers. It is part of the GNU Project. Its name is derived from World Wide Web and get. It supports downloading via HTTP, HTTPS, and FTP protocols.",
+
+    "metadata": {
+        "sourceName": "GNU",
+        "sourceUrl" : "https://www.gnu.org/manual/manual.html"
+    },
+
+    "aliases": [
+        "gnu wget cheat sheet"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/wget.json]: @pa1pal aliases should not contain "cheat sheet". This should just be `gnu wget`

File: share/goodie/cheat_sheets/json/wget.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
+{
+    "id": "wget_cheat_sheet",
+    "name": "Wget",
+    "description": "GNU Wget (or just Wget, formerly Geturl) is a computer program that retrieves content from web servers. It is part of the GNU Project. Its name is derived from World Wide Web and get. It supports downloading via HTTP, HTTPS, and FTP protocols.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/wget.json]: Let's shorten this, it's too long.

"A computer program that retrieves content from web servers via HTTP, HTTPS, and FTP protocols"

**Comment by GuiltyDolphin** [comment]
@pa1pal I'm closing this for now - if you want to finish it up later just let me know and we can re-open it.

**Comment by pa1pal** [comment]
Yes. I want to finish it.
Just did a latest commit with all the changes.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Wget Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/wget_cheat_sheet_mobile.png?y1WuT3eg6D)

**Comment by GuiltyDolphin** [comment]
@pa1pal Thanks :dolphin:

**Comment by daxtheduck** [comment]
@pa1pal this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Wget Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/wget_cheat_sheet_mobile.png?KkH5cNB9dB)

**Pull request merged**